eve/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+version: 0.2
+
+branches:
+  default:
+    stage: pre-merge
+
+models:
+  - Git: &clone
+      name: fetch source
+      repourl: '%(prop:git_reference)s'
+      shallow: True
+      retryFetch: True
+      haltOnFailure: True
+
+stages:
+  pre-merge:
+    worker: &master-worker
+      type: docker
+      path: eve/workers/master
+      volumes:
+        - '/home/eve/workspace'
+    steps:
+      - Git: *clone
+      - ShellCommand:
+          name: install
+          command: npm install
```

Comment:
external deps must be included in the docker image to reduce depedency to outside world during the build

Reply:
There's no more npm install, all set with the docker image